hs_file_types/models/reftimeseries.py

```diff
@@ -313,35 +313,60 @@ class RefTimeseriesFileMetaData(AbstractFileMetaData):
     # field to store the content of the json file (the file that is part
     # of the RefTimeseriesLogicalFile type
     json_file_content = models.TextField()
+    # this is to store abstract
+    abstract = models.TextField(null=True, blank=True)
 
     @property
-    def title(self):
-        """get the title associated with this ref time series"""
+    def has_title_in_json(self):
+        """checks if title is in the uploaded json file"""
+        json_data_dict = self._json_to_dict()
+        return 'title' in json_data_dict['timeSeriesReferenceFile']
+
+    def get_title_from_json(self):
+        """gets the title associated with this ref time series from the json file"""
         json_data_dict = self._json_to_dict()
         if 'title' in json_data_dict['timeSeriesReferenceFile']:
             return json_data_dict['timeSeriesReferenceFile']['title']
         return ''
```

Comment:
@pkdash Seems to me ```has_title_in_json()``` is not needed, but can be achieved by calling ```get_title_from_json()``` instead.

Reply:
@hyi Yes. However, the context where ```has_title_in_json``` has been used I think it makes the code more readable compared to ```get_title_from_json()```.

Reply:
@pkdash I saw how this is used and still think ```has_title_in_json()``` is not needed and will not add much compared to checking whether ```get_title_from_json()``` is empty string or not. You can save ```get_title_from_json()``` returned result to a string variable, and assign other variables with it if it is not empty, which sounds simpler than calling two routines which execute the very similar (logically same) code twice. In any case, I am approving this PR and defer to you on whether you make this change or not.